src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -191,6 +192,48 @@ public static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command)
             return scope;
         }
 
+        public static Scope CreateRabbitMQScope(Tracer tracer, out RabbitMQTags tags, string command, ISpanContext parentContext = null, string spanKind = SpanKinds.Client, DateTimeOffset? startTime = null, string queue = null, string exchange = null, string routingKey = null)
+        {
+            tags = null;
+
+            if (!tracer.Settings.IsIntegrationEnabled(RabbitMQConstants.IntegrationId))
+            {
+                // integration disabled, don't create a scope, skip this trace
```

Comment:
I probably started this trend 😅 , but I see this comment copied everywhere and it should say `...skip this span`.
One day I'll create a PR with a quick search and replace...

Reply:
Perhaps an exercise left for the reader? 😆